src/pages/nfts/[address].tsx

```diff
@@ -420,8 +474,94 @@ export default function NftByAddress({
                         </div>
                       </div>
                     )}
+                    {Boolean(otherListings) &&
+                      otherListings?.map((otherListing, i) => {
+                        const auctionHouseInfo = AUCTION_HOUSE_ADDRESSES.find(
+                          (ah) => ah.address === otherListing?.auctionHouse?.address.toString()
+                        ) ||
+                          MARKETPLACE_PROGRAMS.find(
+                            (ah) => ah.address === otherListing?.marketplaceProgramAddress
+                          ) || {
+                            name: 'Unknown Marketplace',
+                            address: null,
+                            logo: '/images/listings/magiceden.png',
+                            link: 'https://magiceden.io/item-details/',
+                          };
```

Comment:
This looks like a great candidate for a utility function, given where you're using it and how often